src/plugins/credential_management/public/components/edit_credential/edit_credential.tsx

```diff
@@ -72,6 +78,23 @@ export class EditCredentialComponent extends React.Component<
 
     return (
       <EuiPageContent>
+        <div style={{
+          display: "flex",
+          alignItems: "center",
+          justifyContent: "end"
+        }}>
+          <EuiFlexItem>
+            <EuiToolTip content="Remove this credential">
+              <EuiButtonIcon
+                color="danger"
+                onClick={this.removeCredential}
+                iconType="trash"
+                aria-label={"Remove this credential"}
+              // data-test-subj="deleteIndexPatternButton"
+              />
+            </EuiToolTip>
+          </EuiFlexItem>
+        </div>
```

Comment:
Let's use prettier to format the code

https://prettier.io/docs/en/install.html

```
npx prettier --write .
```

Reply:
Got it. Thanks.